library/src/main/scala/scala/scalajs/js/typedarray/TypedArrayBufferOps.scala

```diff
@@ -20,7 +20,7 @@ import java.nio._
  *  for some TypedArrayType, which depends on the type of elements in the
  *  buffer.
  */
-final class TypedArrayBufferOps[
+final class TypedArrayBufferOps[ // scalastyle:ignore
```

Comment:
I think its take is that you shouldn't write type argument lists that are too long ^^